packages/common/src/grpc-operation.js

```diff
@@ -165,6 +165,19 @@ GrpcOperation.prototype.listenForEvents_ = function() {
 };
 
 /**
+ * Wraps the `complete` and `error` events in a Promise.
+ *
+ * @return {promise}
+ */
+GrpcOperation.prototype.promise = function() {
```

Comment:
Since this is a public method, please define this earlier in the file to alpha-sort with the public methods.